playbooks/vars/sample-variables-sap-vm-provision-redhat-ocpv.yml

```diff
@@ -0,0 +1,81 @@
+---
+############################################
+# Red Hat OpenShift Virtualization         #
+############################################
+
+# Namespace where the VM should be created in
+sap_vm_provision_ocp_target_namespace: sap
+
+# Username to be created on guest
+sap_vm_provision_ocp_os_user: minion
+
+# Password for the above user
+sap_vm_provision_ocp_os_user_password:
```

Comment:
Will this allow to run the playbook with an empty string as password?

Reply:
It works with an empty password. Btw. per default password authentication is not used.

Reply:
Added a check so that role fails when password is empty.